src/common/repo.ts

```diff
@@ -0,0 +1,528 @@
+import { rmSync } from 'fs'
+import { pathExists } from 'fs-extra'
+import { rm, writeFile } from 'fs/promises'
+import { globSync } from 'glob'
+import jsonpath from 'jsonpath'
+import { cloneDeep, get, merge, set } from 'lodash'
+import path from 'path'
+import { getDirNames, loadYaml } from './utils'
+import { objectToYaml, writeValuesToFile } from './values'
+
+export const getTeamNames = async (envDir: string): Promise<Array<string>> => {
+  const teamsDir = path.join(envDir, 'env', 'teams')
+  const teamNames = await getDirNames(teamsDir, { skipHidden: true })
+  return teamNames
+}
+
+export interface FileMap {
+  envDir: string
+  kind:
+    | 'AplCoreApp'
+    | 'AplAlerts'
```

Comment:
From my understanding these are plural as we currently use them. But since Kubernetes CRDs usually are defined separately with singular and plural resources, maybe it would be better to indicate explicitly that there is no real singular form of these tuples of things. For example, it could be `AplAlertSet`, `AplBackupSet` etc.